code/modules/mob/living/living_say.dm

```diff
@@ -156,9 +156,13 @@ GLOBAL_LIST_INIT(message_modes_stat_limits, list(
 		language = get_selected_language()
 	var/mob/living/carbon/human/H = src
 	if(!can_speak_vocal(message))
-		if (HAS_TRAIT(src, TRAIT_SIGN_LANG) && H.mind.miming)
-			to_chat(src, "<span class='warning'>You stop yourself from signing in favor of the artform of mimery!</span>")
-			return
+		if(H.mind.miming)
```

Comment:
```suggestion
		if(H.mind?.miming)
```
The integration tests are failing because `mind` doesn't exist for mobs without players.